shared/cluster.go

```diff
@@ -220,9 +220,14 @@ func FetchNodeExternalIPs() []string {
 }
 
 // RestartCluster restarts the service on each node given by external IP.
-func RestartCluster(product, ip string) {
-	_, _ = RunCommandOnNode(fmt.Sprintf("sudo systemctl restart %s*", product), ip)
-	time.Sleep(20 * time.Second)
+func RestartCluster(product, ip string) error {
+	_, err := RunCommandOnNode(fmt.Sprintf("sudo systemctl restart %s*", product), ip)
+	if err != nil {
+		LogLevel("error", "failed to restart %s: on ip: %s %v\n", product, ip, err)
```

Comment:
So we are logging the error and also returning error? Shouldn't we use `ReturnLogError` func here?

Reply:
both works nice but i can update for sure